pontoon/base/static/js/main.js

```diff
@@ -606,7 +606,7 @@ $(function() {
     Pontoon.startLoader();
 
     $.ajax({
-      url: '/api/v1/user/' + $('#server').data('email') + '/',
+      url: '/api/v1/user/' + $('#server').data('uid') + '/',
```

Comment:
I know I suggested otherwise, but I wonder if we should use username here, so that we always use username in URLs. Should prevent crawlers to increment uid and make calls. @jotes?

Anyhow, please make sure that you also update the server data- attribute in the `user_settings.html` template.